server/src/core/symbols/symbol.rs

```diff
@@ -2779,6 +2800,9 @@ impl Symbol {
             if only_fields {
                 content_syms = content_syms.iter().filter(|x| x.borrow().is_field(session)).cloned().collect();
             }
+            if only_methods {
+                content_syms = content_syms.iter().filter(|x| x.borrow().typ() == SymType::FUNCTION).cloned().collect();
```

Comment:
what about a variable that is evaluated to a function definition?

Reply:
Updated!